episodes/files/plans/plan_10.R

```diff
@@ -0,0 +1,42 @@
+options(tidyverse.quiet = TRUE)
+suppressPackageStartupMessages(library(crew))
+source("R/functions.R")
+source("R/packages.R")
+
+# Set up parallelization
+library(crew)
+tar_option_set(
+  controller = crew_controller_local(workers = 2)
+)
+
+tar_plan(
+  # Load raw data
+  tar_file_read(
+    penguins_data_raw,
+    path_to_file("penguins_raw.csv"),
+    read_csv(!!.x, show_col_types = FALSE)
+  ),
+  # Clean data
+  penguins_data = clean_penguin_data(penguins_data_raw),
+  # Build models
+  models = list(
+    combined_model = lm(
+      bill_depth_mm ~ bill_length_mm, data = penguins_data),
+    species_model = lm(
+      bill_depth_mm ~ bill_length_mm + species, data = penguins_data),
+    interaction_model = lm(
+      bill_depth_mm ~ bill_length_mm * species, data = penguins_data)
+  ),
+  # Get model summaries
+  tar_target(
+    model_summaries,
+    glance_with_mod_name_slow(models),
+    pattern = map(models)
+  ),
+  # Get model predictions
+  tar_target(
+    model_predictions,
+    augment_with_mod_name_slow(models),
+    pattern = map(models)
+  )
+)
```

Comment:
A couple of missing terminating newlines in the plans. Setting `"files.insertFinalNewline": true` in VS Code was lifechanging for me

Reply:
Thanks for the tip! Fixed, I hope.